### PR TITLE
add Sphinx config

### PR DESCRIPTION
**Pull request by ajtritt**
@oruebel can you add the logo you made and a favicon version? 

You'll need to uncomment the lines in `conf.py`.

### CODE REVIEW

File: docs/source/conf.py [hunk 1]
```diff
@@ -0,0 +1,340 @@
...
+# "<project> v<release> documentation".
+# html_title = None
+
+# A shorter title for the navigation bar.  Default is the same as html_title.
+# html_short_title = None
+
+# The name of an image file (relative to this directory) to place at the top
+# of the sidebar.
+# html_logo = None
+# html_logo = 'hdmf_logo.png'
```
Comment (ajtritt) [inline docs/source/conf.py]: @oruebel here.

File: docs/source/conf.py [hunk 2]
```diff
@@ -0,0 +1,340 @@
...
+
+# The name of an image file (within the static path) to use as favicon of the
+# docs.  This file should be a Windows icon file (.ico) being 16x16 or 32x32
+# pixels large.
+# html_favicon = 'hdmf_logo-180x180.png'
```
Comment (ajtritt) [inline docs/source/conf.py]: @oruebel and here.

File: docs/source/conf.py (hunk 1 above)
Comment (oruebel) [reply docs/source/conf.py]: I added the logo

File: docs/source/conf.py (hunk 2 above)
Comment (oruebel) [reply docs/source/conf.py]: I added the favicon

**Pull request merged**